app/Models/Campaign.php

```diff
@@ -154,6 +155,20 @@ public function setCauseAttribute($value)
         $this->attributes['cause'] = implode(',', $value);
     }
 
+    /**
+     * Accessor for getting the start_date field.
+     */
+    public function getStartDateAttribute()
+    {
+        $value = $this->attributes['start_date'];
+        if (! $value) {
+            return null;
+        }
+        $date = (new Carbon($value))->format('Y-m-d');
+
+        return new Carbon($date, 'America/New_York');
```

Comment:
It might be nice to add a comment here explaining why we're setting this to EST.